regulations/templates/regulations/preamble-partial.html

```diff
@@ -1,9 +1,9 @@
 {% load macros render_nested %}
 
 <section id="{{full_id}}" class="reg-section" data-page-type="preamble-section" data-doc-id="{{doc_number}}">
-  <ul id="preamble-read">
+  <div id="preamble-read">
     {% render_nested template=sub_template context=sub_context %}
-  </ul>
+  </div>
```

Comment:
I think we'll need to nest an `ol` inside this div -- don't the subtemplates generate `li`s?